system-int-test/test/cmr/system_int_test/search/variable/concept_retrieval_test.clj

```diff
@@ -13,91 +13,168 @@
    [cmr.system-int-test.utils.ingest-util :as ingest]
    [cmr.system-int-test.utils.metadata-db-util :as mdb]
    [cmr.system-int-test.utils.search-util :as search]
-   [cmr.system-int-test.utils.variable-util :as variable]
-   [cmr.transmit.config :as transmit-config]))
+   [cmr.system-int-test.utils.variable-util :as variable]))
 
 (use-fixtures
  :each
  (join-fixtures
-  [(ingest/reset-fixture {"provguid1" "PROV1"
-                          "provguid2" "PROV2"
-                          "provguid3" "PROV3"})
+  [(ingest/reset-fixture {"provguid1" "PROV1"})
    variable/grant-all-variable-fixture]))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
+;; Helper utility functions
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn- get-var
```

Comment:
I was initially not crazy about such a simple helper function, but I think it makes a lot of sense for a test namespace like this where you'll be calling it a bunch, especially when vars are still in such a heavy state of fluctuation.